sys/net/gnrc/pktbuf/gnrc_pktbuf.c

```diff
@@ -109,5 +109,32 @@ gnrc_pktsnip_t *gnrc_pktbuf_reverse_snips(gnrc_pktsnip_t *pkt)
     return reversed;
 }
 
+int gnrc_pktbuf_merge(gnrc_pktsnip_t *pkt)
+{
+    size_t offset = pkt->size;
+    int res = 0;
+
+    size_t size = gnrc_pkt_len(pkt);
+    if (pkt->size >= size) {
```

Comment:
can `pkt->size` be greater than `size`?

Reply:
In the (very rare) edgecase of an integer overflow it might be ;-) but I guess considering the size of the packet buffer (or malloc-space for that matter) this isn't to be expected (and should be an error case in any case). I'll replace with `==`.

Reply:
Done